### DISCUSSION TITLE
DPMSolverMultistepScheduler, how to correctly denoise the specific noise step ?

### DISCUSSION THREAD

**Discussion by HuangChiEn** [huggingface/diffusers#7869] [Q&A]
I'm trying to add the small noise (step t=1) on the latent. Although the document doesn't cover how to add the noise to it, but I found a method called add_noise, which seems fit to my requirements. 
> (I have looked [DPM-solver++ paper](https://arxiv.org/pdf/2211.01095), and the add_noise works exactly match with Eq.1, so I think it can be used to add noise to it). 

---

The issue I faced is that I have no idea how to denoise the noise latent by calling the `scheduler.step(.)` method. 
The following image shows the way I had tried :
![image](https://github.com/huggingface/diffusers/assets/52521165/2642c245-d2cb-43d1-9b86-9e7e9d580b5b)
`sample_latent` is the ground truth (GT), while I calculate the difference between the noise latent with the GT. Then no surprised, the 49-th step noise is the latent with less noise, and i take it as my noise latent.

This is how I call the step method :
`apprx_latent = self.scheduler.step(noise, t, noise_latent, **extra_step_kwargs, return_dict=False)[0]` , where `t=timesteps[0]`. 
> Yeah, I have noticed that it should be `t=timesteps[49]` to consist with the timestep index of noise latent. However, let `t=timesteps[-1]` (which is 49-th step) have the worse results, so I think the first time step is correct.

Unfortunately, the noise latent seems closer to the GT comparing with the denoised latent..
![image](https://github.com/huggingface/diffusers/assets/52521165/0fe1d4e8-668d-45d4-b5b4-f4756d9f2561)

Any suggestion will be appreciated!!
